README.md

```diff
@@ -72,17 +72,42 @@ ENV SPRING_PROFILES_ACTIVE docker,mysql
 In the `mysql section` of the `application.yml` from the [Configuration repository], you have to change 
 the host and port of your MySQL JDBC connection string. 
 
+## Custom metrics monitoring
+
+@todo Add default custom dashboards to grafana
```

Comment:
maybe in a doc/ sub-directory?
I have also to reupload the Spring Petcilinic screenshot of thr readme.md. The link is broken: https://cloud.githubusercontent.com/assets/838318/19653851/61c1986a-9a16-11e6-8b94-03fd7f775bb3.png

Reply:
I agree, screenshots of the default dashboard setup in the meantime until the custom metrics dashboard can be built out and included in a custom image build.

Reply:
For information, `docs/` folder has been introduced in the PR #106